.gitpod.yml

```
@@ -0,0 +1,23 @@
+tasks:
+  - init: |
+      mvn -am -pl war,bom -P quick-build install
```

I wonder if
```suggestion
      mvn -am -pl war,bom -P quick-build clean install
```
works around the extension index problem with the IDE. As noted in https://github.com/jenkinsci/jenkins/pull/6590#pullrequestreview-992504596 a clean build _after_ failure works.